.github/workflows/build.yml

```diff
@@ -188,6 +251,68 @@ jobs:
           cd wrappers/golang
           go run cmd/demo/demo.go
 
+  build-javascript:
+    name: Build and Test JavaScript wrapper
+    needs: [build-manylinux, build-release]
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: wrappers/javascript
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node.JS 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Fetch library artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: library-${{ runner.os }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build JavaScript Wrapper
+        run: yarn build
+
+      - name: Lint JavaScript Wrapper
+        run: yarn lint
+
+      - name: Format Check JavaScript Wrapper
+        run: yarn check-format
+
+      - name: Type Check JavaScript Wrapper
+        run: yarn check-types
+
+      - name: Test JavaScript Wrapper
+        env:
+          # binary is downloaded to root of repository
+          LIB_INDY_VDR_PATH: ../../../
+        run: yarn test
+
+      - name: Set NPM config
+        if: |
+          github.event_name == 'release' || 
+          (github.event_name == 'workflow_dispatch' && github.event.inputs.publish-wrappers == 'true')
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" >> .npmrc
+          echo "registry=https://registry.npmjs.org/" >> .npmrc
+          echo "always-auth=true" >> .npmrc
+
+      # TODO: this will currently publish the version as defined in the package.json. If the version already
+      # exists on NPM it will skip the publishing. This means if a new version is released, but the version hasn't been
+      # updated in the package.json files yet, it won't publish to NPM (which is kinda nice). We should however add a check
+      # to see if the JS version matches the release version (do we want to keep the js and rust version in sync?)
```

Comment:
We have discussed this externally as well, but my preference goes to no version sync. We will likely create many many patches when we integrate with AFJ and it would be I think rather annoying for the other packages. 

We can create a simple mapping in the README and we have to make sure that the latest indy-vdr is supported by the latest wrapper in JS.

Reply:
Agreed, will update the command.